nss-day-cohort-48/rare-client-polite-pirates
# Jf register

#### Changes Made

1. Added file `PULL_REQUEST_TEMPLATE.MD` to `src` directory.
2. Modified file `Register.js` to include the keys `bio` `profile_image_url` `created_on` and `active`, also changed the keys format to snake case.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout jf-register
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
  

- [x] open both the SERVER side and the CLIENT side in your code. The server side can be main.
- [x] Run the debugger
- [x] Npm start on the Client side branch jf-register
- [x] Click "Not a member yet"
- [x] Fill out form
- [x] Submit form and check the DEV tool network tab for a 200 & 201 fetch
- [x] Check Postman and GET all users to see if your new user was submitted

4. View code file.
   > Confirm file modifications are present as indicated above.
